presto-main/src/main/java/io/prestosql/execution/scheduler/TopologyAwareNodeSelectorConfig.java

```diff
@@ -48,4 +53,29 @@ public TopologyAwareNodeSelectorConfig setLocationSegmentNames(String locationSe
                 .splitToList(locationSegmentNames);
         return this;
     }
+
+    public String getNetworkTopologyFile()
```

Comment:
Instead of putting these properties in this config, I think we should do conditional bindings. We can have a `node-scheduler.network-topology.type` which a users uses to switch between the default `FLAT` topology and the `FILE` topology loader.  Then these properties would be in a new config class, so they only appear with `FILE` is set.